src/SlamData/Workspace/Card/BuildChart/PivotTable/Component.purs

```diff
@@ -0,0 +1,443 @@
+
+{-
+Copyright 2016 SlamData, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-}
+
+module SlamData.Workspace.Card.BuildChart.PivotTable.Component where
+
+import SlamData.Prelude
+
+import Data.Argonaut as J
+import Data.Array as Array
+import Data.Int (toNumber)
+
+import CSS as C
+import Halogen as H
+import Halogen.Component.Utils.Drag as Drag
+import Halogen.HTML.Indexed as HH
+import Halogen.HTML.Events.Indexed as HE
+import Halogen.HTML.Properties.Indexed as HP
+import Halogen.HTML.Properties.Indexed.ARIA as ARIA
+import Halogen.HTML.CSS.Indexed as HC
+
+import SlamData.Form.Select as S
+import SlamData.Monad (Slam)
+import SlamData.Workspace.Card.BuildChart.Aggregation as Ag
+import SlamData.Workspace.Card.BuildChart.DimensionPicker.Component as DPC
+import SlamData.Workspace.Card.BuildChart.PivotTable.Component.ChildSlot as PCS
+import SlamData.Workspace.Card.BuildChart.PivotTable.Component.Query (Query(..), QueryC)
+import SlamData.Workspace.Card.BuildChart.PivotTable.Component.State (State, Selecting(..), modelFromState, stateFromModel, initialState, reorder, setColumnAggregation)
+import SlamData.Workspace.Card.BuildChart.PivotTable.Model (Column(..))
+import SlamData.Workspace.Card.CardType as CT
+import SlamData.Workspace.Card.CardType.ChartType as CHT
+import SlamData.Workspace.Card.Common.Render (renderLowLOD)
+import SlamData.Workspace.Card.Component as CC
+import SlamData.Workspace.Card.Model as Card
+import SlamData.Workspace.Card.Port as Port
+import SlamData.Workspace.LevelOfDetails (LevelOfDetails(..))
+
+type DSL = H.ParentDSL State PCS.ChildState QueryC PCS.ChildQuery Slam PCS.ChildSlot
+
+type HTML = H.ParentHTML PCS.ChildState QueryC PCS.ChildQuery Slam PCS.ChildSlot
+
+pivotTableBuilderComponent ∷ CC.CardComponent
+pivotTableBuilderComponent = CC.makeCardComponent
+  { cardType: CT.ChartOptions CHT.PivotTable
+  , component: H.parentComponent
+      { render
+      , eval: coproduct evalCard evalOptions
+      , peek: Just peek
+      }
+  , initialState: H.parentState initialState
+  , _State: CC._BuildPivotTableState
+  , _Query: CC.makeQueryPrism' CC._BuildPivotTableQuery
+  }
+
+render ∷ State → HTML
+render st =
+  if st.levelOfDetails ≡ High
```

Comment:
Well then I guess I should have remembered to add LOD 😆 

Why do you say this if/else doesn't work?